components/src/histogram.tsx

```diff
@@ -19,28 +19,31 @@
  *
  */
 
-import { getBooleanAttributes } from './utils';
+import { useEffect } from 'react';
+import { getBooleanAttributes, setElementStyles } from './utils';
 
 function IobioHistogram({
 	brokerKey,
-	color,
+	styles,
 	ignoreOutliers = false,
 	title,
 }: {
 	brokerKey: string;
-	color?: string;
+	styles?: string;
 	ignoreOutliers?: boolean;
 	title?: string;
 }) {
+	useEffect(() => {
+		const selector = `iobio-histogram[broker-key=${brokerKey}]`;
+		const element = document.querySelector(selector);
+
+		if (element && styles) {
+			setElementStyles(element, styles);
+		}
+	}, []);
+
```

Comment:
Keep in mind there could be multiple elements eg. multiple histograms on a page. nothing stopping two histograms with the same `brokerKey` existing. also `document.querySelector` will always return only 1 element. So there's some fragileness to this. 

This hook only accounts for onMount, there's no dependency, when in reality this is introducing `styles` as a dependency of the hook. Nothing stopping a consumer from dynamically changing the `styles` prop.

Reply:
Yeah selector is one thing I agree can be improved and adding styles as dependency

What is the issue with `document.querySelector will always return only 1 element` ?

Reply:
it's dependant on having a selector for an individual element. 
in this case if there are two histogram with the same broker key and you want to style the 2nd one, it won't work.